code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -198,20 +197,20 @@ var/global/list/image/splatter_cache=list()
 
 
 /obj/effect/decal/cleanable/blood/gibs/proc/streak(var/list/directions)
-        spawn (0)
-                var/direction = pick(directions)
-                for (var/i = 0, i < pick(1, 200; 2, 150; 3, 50; 4), i++)
-                        sleep(3)
-                        if (i > 0)
-                                var/obj/effect/decal/cleanable/blood/b = new /obj/effect/decal/cleanable/blood/splatter(src.loc)
-                                b.basecolor = src.basecolor
-                                b.update_icon()
-                                for(var/datum/disease2/disease/D in src.viruses)
-                                        var/datum/disease2/disease/ND = D.getcopy()
-                                        b.viruses += ND
-
-                        if (step_to(src, get_step(src, direction), 0))
-                                break
+	set waitfor = 0
```

Comment:
it's copypasted from -tg- due to lazyness, i can replace it with a spawn, but it probably doesn't matter given that I don't think anything calls this rapidly